ext/ftp/tests/ftp_site_basic.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+ftp_site function basic functionality
+--CREDITS--
+Gabriel Caruso (carusogabriel34@gmail.com)
+--SKIPIF--
+<?php require 'skipif.inc'; ?>
+--FILE--
+<?php
+require 'server.inc';
+
+$ftp = ftp_connect('127.0.0.1', $port);
+$ftp or die("Couldn't connect to the server");
+
+ftp_login($ftp, 'user', 'pass') or die("Couldn't login into the server");
+
+var_dump(ftp_site($ftp, 'CHMOD 0600 file'));
+var_dump(ftp_site($ftp, 'foo bar baz'));
+?>
+--EXPECTF--
```

Comment:
Very different from what I get locally https://travis-ci.org/php/php-src/jobs/334022903#L1974 :thinking:

Reply:
Is this supposed to return `SITE foo bar baz`, the command from `ftp_site`?